diff -urN iPodder-linux.org/iPodderGui.py iPodder-linux/iPodderGui.py
--- iPodder-linux.org/iPodderGui.py	2005-07-26 07:27:35.000000000 +0200
+++ iPodder-linux/iPodderGui.py	2005-07-30 16:12:46.763549288 +0200
@@ -70,7 +70,7 @@
 TIMER_INTERVAL = 10000
 
 #Version check
-CHECK_VERSION = 2.1.1
+CHECK_VERSION = "2.1.1"
 
 #GUI params -- Don't edit me.
 ID_EXIT = 101
diff -urN iPodder-linux.org/iPodder.py.rej iPodder-linux/iPodder.py.rej
--- iPodder-linux.org/iPodder.py.rej	2005-07-25 00:29:56.000000000 +0200
+++ iPodder-linux/iPodder.py.rej	1970-01-01 01:00:00.000000000 +0100
@@ -1,53 +0,0 @@
-***************
-*** 109,114 ****
-              return "Finished"
-          if self.status == "partial":
-              return "Partially downloaded"
-          log.debug("Enclosure has unknown status %s" % self.status)
-          return "Unknown"
-      
---- 109,116 ----
-              return "Finished"
-          if self.status == "partial":
-              return "Partially downloaded"
-+         if self.status == "error":
-+             return "Error"
-          log.debug("Enclosure has unknown status %s" % self.status)
-          return "Unknown"
-      
-***************
-*** 173,178 ****
-              self.grabber()
-          except grabbers.GrabError, ex: 
-              self.error("Can't grab %s: %s", str(feedinfo), ex.message)
-              return
-          self.grabber = None
-          if self.abort: 
---- 175,181 ----
-              self.grabber()
-          except grabbers.GrabError, ex: 
-              self.error("Can't grab %s: %s", str(feedinfo), ex.message)
-+             feedinfo.status = "error"
-              return
-          self.grabber = None
-          if self.abort: 
-***************
-*** 788,794 ****
-              if not self.quitting and not torrentfile:
-                  self.m_downloadlog.append(filename)
-          except grabbers.GrabError, ex: 
-              log.error("Can't grab %s: %s", str(enc), ex.message)
-          except:
-              log.exception("Can't grab %s", enc)
-          encinfo.grabber = None
---- 791,800 ----
-              if not self.quitting and not torrentfile:
-                  self.m_downloadlog.append(filename)
-          except grabbers.GrabError, ex: 
-+             # Generic error but file not downloaded, set status
-              log.error("Can't grab %s: %s", str(enc), ex.message)
-+             encinfo.status = "error"
-+             return False, None
-          except:
-              log.exception("Can't grab %s", enc)
-          encinfo.grabber = None
